File: sdkwork-app-sdk-go/types/shop_detail_vo.go
package types

// 店铺详情响应
type ShopDetailVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Name string `json:"name"`
	Description string `json:"description"`
	LogoUrl string `json:"logoUrl"`
	CoverUrl string `json:"coverUrl"`
	ContactPhone string `json:"contactPhone"`
	ContactEmail string `json:"contactEmail"`
	Address string `json:"address"`
	Latitude float64 `json:"latitude"`
	Longitude float64 `json:"longitude"`
	LicenseNumber string `json:"licenseNumber"`
	Tags []string `json:"tags"`
	BusinessHours string `json:"businessHours"`
	Status string `json:"status"`
	StatusName string `json:"statusName"`
	OwnerId string `json:"ownerId"`
	OwnerName string `json:"ownerName"`
	ProductCount int `json:"productCount"`
	OrderCount int `json:"orderCount"`
	Rating float64 `json:"rating"`
	FollowerCount int `json:"followerCount"`
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/shop_detail_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ShopDetailVO:
    """店铺详情响应"""
    created_at: str = None
    updated_at: str = None
    name: str = None
    description: str = None
    logo_url: str = None
    cover_url: str = None
    contact_phone: str = None
    contact_email: str = None
    address: str = None
    latitude: float = None
    longitude: float = None
    license_number: str = None
    tags: List[str] = None
    business_hours: str = None
    status: str = None
    status_name: str = None
    owner_id: str = None
    owner_name: str = None
    product_count: int = None
    order_count: int = None
    rating: float = None
    follower_count: int = None
